File: hot-framework/src/test/resources/be/icode/hot/test/nio/python/https-client-selfsigned.py
options = {
	'hostname': 'localhost',
	'port': 8443,
	'path': "/test",
	'rejectUnauthorized':False
}

class Body:
	pass
	
def responseHandler(response):

    print "connected"
    body = Body()
    body.content = ""
	
    def dataHandler (chunck):
        body.content += chunck

    def endHandler ():
        print body.content
        print response.statusCode

    response.on("data",dataHandler)
    response.on("end", endHandler)

request = client.request (options , responseHandler)
request.end()

File: hot-framework/src/test/resources/be/solidx/hot/test/nio/js/https-client-selfsigned.js
var options = {
	hostname: 'localhost',
	port: 8443,
	path: "/test",
	rejectUnauthorized:false,
	
}

var request = client.request(options, function(response) {
	var body = ""
	response.on('data', function(chunck) {
		body+=chunck
	})
	
	response.on('end', function() {
		hprint(body)
		hprint(response.statusCode)
	})
})

request.on('error', function(message) {
	hprint(message)
})

request.end() 